topgun/k8s/k8s_suite_test.go

```diff
@@ -139,12 +269,13 @@ func helmDeploy(releaseName, namespace, chartDir string, args ...string) *gexec.
 
 func helmInstallArgs(args ...string) []string {
 	helmArgs := []string{
-		"--set=web.livenessProbe.failureThreshold=3",
-		"--set=web.livenessProbe.initialDelaySeconds=3",
-		"--set=web.livenessProbe.periodSeconds=3",
-		"--set=web.livenessProbe.timeoutSeconds=3",
 		"--set=concourse.web.kubernetes.keepNamespaces=false",
+		"--set=concourse.worker.bindIp=0.0.0.0",
```

Comment:
this is necessary for tests where we make direct requests to `gdn` - `kubectl port-forward` can forward to loopback addresses (127.0.0.1/8), but that's not true with a direct request to an iface.